<!--
 Copyright (c) 2021  IBM Corporation

 Permission is hereby granted, free of charge, to any person obtaining a copy
 of this software and associated documentation files (the "Software"), to deal
 in the Software without restriction, including without limitation the rights
 to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
 copies of the Software, and to permit persons to whom the Software is
 furnished to do so, subject to the following conditions:

 The above copyright notice and this permission notice shall be included in all
 copies or substantial portions of the Software.

 THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
 IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
 FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
 AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
 LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 SOFTWARE.
-->

<launch>
  <!-- deprecated -->
  <!--
    This file is kept for backward compartibility.
    use "$(find mf_localization)/launch/multi_floor_2d_rss_localization.launch"
  -->
  <arg name="map_config_file"/>
  <arg name="beacons_topic" default="beacons"/>
  <arg name="with_odom_topic" default="false"/>

  <arg name="points2_topic" default="velodyne_points" />
  <arg name="imu_topic" default="imu/data" />
  <arg name="odom_topic" default="odom"/>

  <include file="$(find mf_localization)/launch/multi_floor_2d_rss_localization.launch">
    <arg name="map_config_file" value="$(arg map_config_file)"/>
    <arg name="beacons_topic" value="$(arg beacons_topic)"/>
    <arg name="with_odom_topic" value="$(arg with_odom_topic)"/>

    <arg name="points2_topic" value="$(arg points2_topic)"/>
    <arg name="imu_topic" value="$(arg imu_topic)"/>
    <arg name="odom_topic" value="$(arg odom_topic)"/>
  </include>
</launch>
